mkdocs.yml

```diff
@@ -112,6 +112,9 @@ nav:
       - Software Collaboration with GitHub and GitLab: workflow_solutions/git_collaboration.md
   - Cheaha Guide:
       - Getting Started: cheaha/getting_started.md
+      - Tutorials: 
+          - cheaha/tutorial/index.md
+          - Anaconda Environment Tutorial for PyTorch and TensorFlow: cheaha/tutorial/pytorch_tensorflow.md
```

Comment:
Let's shorten the title. I think we can remove `for PyTorch and TensorFlow`. If we create more tutorials, we can find a way to distinguish among them later.